airbyte-lib/examples/run_faker.py

```diff
@@ -12,14 +12,19 @@
 import airbyte_lib as ab
 
 
-SCALE = 5_000_000  # Number of records to generate between users and purchases.
+SCALE = 500_000  # Number of records to generate between users and purchases.
 
+# This is a dummy secret, just to test functionality.
+DUMMY_SECRET = ab.get_secret("DUMMY_SECRET")
```

Comment:
this does not seem to be used anywhere.

Reply:
Yes, that's correct. It's literally just there to make sure our progress printer isn't conflicting with our ability to take input from the user, which was previously an issue.